scheduler/src/main/java/org/apache/mesos/elasticsearch/scheduler/TaskInfoFactory.java

```diff
@@ -70,6 +70,7 @@
                 .setContainer(Protos.ContainerInfo.newBuilder()
                         .setType(Protos.ContainerInfo.Type.DOCKER)
                         .setDocker(Protos.ContainerInfo.DockerInfo.newBuilder().setImage("mesos/elasticsearch-executor"))
+                        .addVolumes(Protos.Volume.newBuilder().setContainerPath("/data").setHostPath("/var/lib/elasticsearch").setMode(Protos.Volume.Mode.RW).build())
```

Comment:
Will mesos always have permission to RW /data externally? And will /var/lib/elasticsearch always exist?